packages/main/src/themes/Toast.css

```diff
@@ -0,0 +1,84 @@
+.ui5-toast-root-parent {
+	position: fixed;
+	left: 0;
+	right: 0;
+	top: 0;
+	bottom: 0;
+	width: 100%;
+	display: none;
+	height: 100%;
+	pointer-events: none;
+	overflow: hidden;
+	z-index: 999;
+	justify-content: center;
+	align-items: flex-end;
+}
+
+.ui5-toast-text {
+	font-family: var(--sapUiFontFamily);
```

Comment:
We always try to give our users control their appearance as possible. 

When these styles are set to element inside the shadow DOM, the user can`t alter them. 
```css
font-family: var(--sapUiFontFamily);
color: var(--sapUiListTextColor);
font-size: var(--sapMFontMediumSize);
```

For example, this is not possible currently
```css
ui5-toast {
    color: red;
    font-size: 2rem;
}
```

To enable this, put the styles on the host, then just inherit them where you need:

```css
:host {
	font-family: var(--sapUiFontFamily);
	color: var(--sapUiListTextColor);
	font-weight: normal;
	font-size: var(--sapMFontMediumSize);
}

.ui5-toast-text {
	color: inherit;
	font: inherit;
    ---
}
```